packages/synapse-react-client/src/components/AccountLevelBadge/AccountLevelBadge.tsx

```diff
@@ -1,67 +1,66 @@
+import { Link, Tooltip, Typography } from '@mui/material'
 import React from 'react'
 import { ReactComponent as Certified } from '../../assets/icons/account-certified.svg'
-import { ReactComponent as Registered } from '../../assets/icons/account-registered.svg'
 import { ReactComponent as Validated } from '../../assets/icons/account-validated.svg'
-import { SynapseConstants } from '../../utils'
-import { useGetUserBundle } from '../../synapse-queries/user/useUserBundle'
-import { ErrorBanner } from '../error/ErrorBanner'
 
-export type AccountLevelBadgeProps = {
-  userId: string
+export type AccountLevelBadgeType = 'certified' | 'validated'
+type AccountLevelBadgeConfig = {
+  label: string
+  description: string
+  tooltipText: string
+  icon: React.ReactElement
 }
 
-export const accountLevelRegisteredLabel: string = 'Registered'
-export const accountLevelCertifiedLabel: string = 'Certified'
-export const accountLevelVerifiedLabel: string = 'Validated'
+export const accountLevelBadgeConfig: Record<
+  AccountLevelBadgeType,
+  AccountLevelBadgeConfig
+> = {
+  certified: {
+    label: 'Certified',
+    description: 'Has access to normal functionality.',
+    tooltipText:
+      'This user has passed Sage Certification, which means they can upload data and create new projects and tables.',
+    icon: <Certified />,
+  },
+  validated: {
+    label: 'Validated',
+    description: 'Identity has been manually verified.',
+    tooltipText:
+      'This user is Validated. Their identity and credentials have been manually verified by Sage Bionetworks.',
+    icon: <Validated />,
+  },
+}
 
-const CERTIFICATION_AND_VERIFICATION_BUNDLE_MASK =
-  SynapseConstants.USER_BUNDLE_MASK_IS_CERTIFIED |
-  SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED
+export type AccountLevelBadgeProps = {
+  badgeType: AccountLevelBadgeType
+}
 
 export const AccountLevelBadge: React.FunctionComponent<
   AccountLevelBadgeProps
-> = ({ userId }: AccountLevelBadgeProps) => {
-  const {
-    data: userBundle,
-    isLoading,
-    error,
-  } = useGetUserBundle(userId, CERTIFICATION_AND_VERIFICATION_BUNDLE_MASK)
```

Comment:
Moved the service call into AccountLevelBadges. This component now just creates a particular type of badge.